app/models/concerns/statusable.rb

```diff
@@ -19,7 +19,7 @@ module Statusable
                  help_text: I18n.t('patient.status.help.fulfilled')},
     dropoff: { key: I18n.t('patient.status.key.dropoff'),
                help_text: I18n.t('patient.status.help.dropoff')},
-    resolved: { key: I18n.t('patient.status.key.resolved', fund: FUND),
+    resolved: { key: I18n.t('patient.status.key.resolved', fund: ActsAsTenant.current_tenant&.name),
```

Comment:
Using the lonely operator here because tenant isn't resolved on startup, just on request. Need to check that it works properly before bringing out of draft.

Reply:
sure does! 
![image](https://user-images.githubusercontent.com/3866868/135778062-04325fb0-f0bb-4de5-a5e8-9991737a6e76.png)